### PR TITLE
Create calculator component 

### PR DESCRIPTION
**Pull request by sheylaPozo**
# Components:

- Create Calculator class component:
> components >> Calculator.js file, to render the calculator layout.

# CSS:
-Add basic styling to the calculator.

## Thank you for taking the time to review my PR! ✨🎠😊

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/*eslint-disable */
```
Comment (vichuge) [inline src/components/Calculator.js]: - [x] It's understandable if you have one or two eslint rules that you don't know how to fix. But disable all eslint I think is not appropriate, I invite you to quit it and fix all issues that you could have. Eslint is important because this helps you to get a better and more professional code, at the beginning could be hard to follow it, but you'll see how you improve your code following this.
- Hint: If you install everything for [linters config locally](https://github.com/microverseinc/linters-config/tree/master/react-redux) you could use `npx eslint .` to see the mistakes and then use `npx eslint . --fix` to fix the most. Please, consider this, is the easiest way to do it.
image

File: src/App.js [hunk 2]
```diff
@@ -1,29 +1,11 @@
-import logo from './logo.svg';
-import './App.css';
+/*eslint-disable */
```
Comment (vichuge) [inline src/App.js]: - [x] It's understandable if you have one or two eslint rules that you don't know how to fix. But disable all eslint I think is not appropriate, I invite you to quit it and fix all issues that you could have. Eslint is important because this helps you to get a better and more professional code, at the beginning could be hard to follow it, but you'll see how you improve your code following this.
- Hint: If you install everything for [linters config locally](https://github.com/microverseinc/linters-config/tree/master/react-redux) you could use `npx eslint .` to see the mistakes and then use `npx eslint . --fix` to fix the most. Please, consider this, is the easiest way to do it.
image

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (vichuge) [reply src/components/Calculator.js]: - Hello again, talking specifically to the `react/prefer-stateless-function` it's ok if you create an exception on the script, but for the others, please, work on it. Sorry if I don't say this before. 🐱

File: src/App.js (hunk 2 above)
Comment (vichuge) [reply src/App.js]: - Hello again, talking specifically to the `react/prefer-stateless-function` it's ok if you create an exception on the script, but for the others, please, work on it. Sorry if I don't say this before. 🐱

File: src/components/Calculator.js (hunk 1 above)
Comment (sheylaPozo) [reply src/components/Calculator.js:1]: > ok working on it

File: src/App.js (hunk 2 above)
Comment (sheylaPozo) [reply src/App.js:1]: working on it

File: src/components/Calculator.js (hunk 1 above)
Comment (sheylaPozo) [reply src/components/Calculator.js]: - corrected

File: src/App.js (hunk 2 above)
Comment (sheylaPozo) [reply src/App.js]: > corrected

File: src/components/Calculator.js (hunk 1 above)
Comment (sheylaPozo) [reply src/components/Calculator.js]: > corrected

**Review by bellom — APPROVED** [review]
## Hi @sheylaPozo  👏  

**Great work making all the necessary corrections from the previous review.**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sheylaPozo** [comment]
thank you :)
